src/cclib/io/ccio.py

```diff
@@ -276,26 +256,53 @@ def ccopen(source, *args, **kargs):
             inputfile.seek(0, 0)
         if not is_stream:
             inputfile.close()
-            return filetype(source, *args, **kargs)
-        return filetype(inputfile, *args, **kargs)
+            return filetype(source, *args, **kwargs)
+        return filetype(inputfile, *args, **kwargs)
 
 
 def fallback(source):
     """Attempt to read standard molecular formats using other libraries.
 
-    Currently this will read XYZ files with OpenBabel, but this can easily
-    be extended to other formats and libraries, too.
+    Currently this will read files with Open Babel, but this can
+    easily be extended to other formats and libraries, too.
     """
 
     if isinstance(source, str):
         ext = os.path.splitext(source)[1][1:].lower()
         if _has_cclib2openbabel:
-            if ext in ('xyz', ):
+            import pybel as pb
```

Comment:
Is pybel always available if python-openbabel is installed?

Reply:
Yes.